### ISSUE TITLE
BUG: Image back-linking and renaming not supported

### ISSUE THREAD

**Issue by draganmarjanovic** [vertis/obsidian-image-layouts#19]
## Behaviour:
A. When viewing an image in the vault, if it is referenced / loaded via the plugin, it is not possible to see the backlinks (ie. where the document is used).
B. As there is no link established, when renaming an image, Obsidian does not prompt to update the references within the vault.

## Expected Behaviour:
A. When viewing an image, I can see all the locations in which it is used / referenced.
B. When updating the name of an image, all the references to it are also updated.

**Comment by draganmarjanovic** [comment]
This actually seems non trivial to fix...

There's a few (contentious) discussion threads, linking them here for reference:
- https://forum.obsidian.md/t/linked-mentions-in-codeblocks-showing-as-unlinked/21679
- https://forum.obsidian.md/t/do-not-show-x-in-unlinked-mentions-avoid-backlinks-detection-in-tags-wikilinks-and-md-links/8599
- https://www.reddit.com/r/ObsidianMD/comments/ut2ioa/i_want_to_make_links_to_text_blocks_inside_of_my/

---

The only two options I see here are:
- Accept the issue / don't fix
- Investigate using something other than code blocks to trigger and generate the layouts

**Comment by vertis** [comment]
Yeah, it's not easy to solve unfortunately. I'm going to close this for now because it's not going to be possible for me to solve this.
